polaris-react/src/components/DataTable/components/Navigation/Navigation.tsx

```diff
@@ -44,7 +46,12 @@ export function Navigation({
   );
 
   return (
-    <div className={styles.Navigation}>
+    <div
+      className={classNames(
+        styles.Navigation,
+        increasedTableDensity && styles.IncreasedTableDensity,
```

Comment:
Maybe we move `styles.IncreasedTableDensity,` to the tableWrapperClasses so that it falls outside of navigation and we only have the once spot for it.

Reply:
Then we don't need to pass things down to this component.

Reply:
👍